openfisca_core/tools/test_runner.py

```diff
@@ -230,26 +231,35 @@ def _run_test(period_str, test, verbose = False, options = {}):
     scenario.suggest()
     simulation = scenario.new_simulation(trace = verbose)
     output_variables = test.get(u'output_variables')
+    tested_variables = output_variables.keys()
+    if test_only is not None:
+        tested_variables = test_only
     if output_variables is not None:
         try:
+            if test_only is not None:
+                for variable in test_only:
+                    assert variable in output_variables.keys(), "La variable {} n'est pas présente dans les outputs variables du test".format(variable)
```

Comment:
Any reason to throw an error in that case ? It seems limitating : I cannot run all the `rsa` tests in mes-aides  for instance, as an error would be raised for files that don't test `rsa`.

Reply:
@bfabre01 @ClaireLeroyIPP :  
 - should we raise an error to force the user to be sure to list the right tests to test his specific variable ?
 - should we ignore tests when the variable is not present (my favorite)  ?